### DISCUSSION TITLE
Add option to exclude chunk upload routes from Throttle middleware

### DISCUSSION THREAD

**Discussion by fusigabs** [rahulhaque/laravel-filepond#42] [Ideas]
When doing chunk uploads, it may be necessary to disable the throttle middleware for the `patch` route since several requests may need to be sent to the server for the chunks. Something like:

```php
Route::patch(config('filepond.server.url', '/filepond'), [config('filepond.controller', FilepondController::class), 'patch'])
->withoutMiddleware('throttle')
->name('filepond-patch');
```
This can be a config entry to allow users the option to disable this middleware or not.

**Comment by rahulhaque** [comment]
@nfunwigabga did you try after overriding the specific route in your `routes/web.php`?

**Reply by fusigabs** [reply]
Yes absolutely. In my case, I am using api routes, so my definition was 
```php
->withoutMiddleware('throttle:api');
```
and this works.

**Reply by rahulhaque** [reply]
I would recommend you keep it this way instead of me adding another option in the config file. By overriding, you have more control over each of the filepond routes than specific option for just chunk upload route.

Thank you for using my package.
